frontends/ol-utilities/src/components/embedly/util.ts

```diff
@@ -82,9 +82,9 @@ const createStylesheet = (doc: Document, css: string) => {
 }
 
 const getEmbedlyKey = (): string | null => {
-  const key = window.SETTINGS?.embedlyKey
+  const key = APP_SETTINGS.embedlyKey
   if (typeof key === "string") return key
-  console.warn("window.SETTINGS.EMBEDLY_KEY should be a string.")
+  console.warn("APP_SETTINGS.EMBEDLY_KEY should be a string.")
```

Comment:
It's `APP_SETTINGS. embedlyKey` now, right? But if you want to SHOUT CASE the keys, that's ok, too.